kernel/packages/unity-interface/UnityInterface.ts

```diff
@@ -155,14 +156,21 @@ export class UnityInterface {
     this.SendMessageToUnity('Main', 'DumpRendererLockersInfo')
   }
 
-  public CreateUIScene(data: { id: string; baseUrl: string }) {
+  public CreateGlobalScene(data: {
+    id: string;
+    name: string;
+    baseUrl: string,
+    contents: Array<ContentMapping>,
+    icon?: string,
+    isPortableExperience: boolean,
```

Comment:
Just curious, what does this change?

Reply:
@sandrade-dcl do you know that?

Reply:
Now, with the introduction of the "Portable Experiences" concept, a global scene can be created as a "normal global scene" (`isPortableExperience = false`) or as a "portable experience" (`isPortableExperience = true`). When the global scene is a PE, we need also to send to Unity some extra info in order to create the corresponding icon in the taskbar. These extra fields are basically: the name of the experience (it will be loaded as a tooltip in the taskbar button) and the icon with its mapping (it will be loaded as image in the taskbar button).